src/AppBundle/Entity/Repository/DepartmentRepository.php

```diff
@@ -31,6 +31,14 @@ public function findDepartmentById($id)
         return $departments;
     }
 
+    public function findAllWithActiveAdmission()
+    {
+        return array_filter($this->findAll(), function (Department $department) {
```

Comment:
Går det raskere å filtrere arrayet enn å gjøre et query?

Reply:
Det går nok raskere å gjøre et query, men da er det snakk om <1ms. Brukte filter siden ble mye enklere kode enn lage en query for å joine med semester og sjekke at `nå` er innenfor opptaksperioden (dette er jo allerede implementert i `$semester->hasActiveAdmission()`.

Reply:
Da går det fint for min del :+1: